.husky/pre-commit-phpcbf.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+
+# Disable exit on error since we handle exit codes manually
+set +e
+
+# Get list of staged PHP files
+STAGED_PHP_FILES=$(git diff --cached --name-only --diff-filter=d | grep ".php$") || true
+
+# Exit if no PHP files are staged
+if [ -z "$STAGED_PHP_FILES" ]; then
+    exit 0
+fi
+
+echo "Running PHPCBF on staged files..."
+
+# First try to fix what we can with PHPCBF
+echo "$STAGED_PHP_FILES" | xargs vendor/bin/phpcbf
+
+echo 'does this run?'
+
+# Check the return code
+PHPCBF_STATUS=$?
```

Comment:
This is going to get the status of the `echo` on line 19, not the `xargs` on line 17. Move it up.

Reply:
Fixed by removing the debug `echo` d1afd5f - thanks.